libraries/provider_repo.rb

```diff
@@ -27,6 +27,7 @@ class NetuitiveCookbook::NetuitiveRepoProvider < Chef::Provider::LWRPBase
       include_recipe 'apt'
       # add repo and key
       apt_repository 'netuitive' do
+        distribution ''
```

Comment:
I need to evaluate this against lots of debians so I am hesitant to merge and release this without a lot of testing and research. If someone is blocked they can update their berksfile to use this branch.